website/content/docs/templates/hcl_templates/blocks/source.mdx

```diff
@@ -12,29 +12,38 @@ The top-level `source` block defines reusable builder configuration blocks:
 
 `@include 'from-1.5/sources/example-block.mdx'`
 
+The first label — `happycloud` here — is the builder type. The second label is
+the name or identifier you want to give to the source. There can be only one
```

Comment:
For people just skimming the docs I think it might be helpful to say that it's unique right in the beginning. 
```suggestion
the unique name or identifier you want to give to the source. There can be only one
```